uberfire-structure/uberfire-structure-backend/src/main/java/org/guvnor/structure/backend/organizationalunit/OrganizationalUnitServiceImpl.java

```diff
@@ -495,6 +500,10 @@ public Boolean isValidGroupId(final String proposedGroupId) {
         return false;
     }
 
+    public void onAfterDeleteOrganizationalUnitEvent(@Observes AfterDeleteOrganizationalUnitEvent afterDeleteOrganizationalUnitEvent) {
```

Comment:
@ederign could you check this? because is empty. It works, but well... I'm not totally sure about this

Reply:
@adrielparedes this is the only way to propagate client-side events to backend.

Maybe we should add this cases for: ClusterLibraryObserver.java

wdyt?